### ISSUE TITLE
Request your work to be included in the awesome Zepp OS project

### ISSUE THREAD

**Issue by MyoungXUE** [Alpaca131/TransitTimeViewer#1]
Hello @Alpaca131

Please allow me to introduce myself, I am Myoung XUE, an engineer from Zepp OS Team.

We have an awesome project https://github.com/zepp-health/awesome-zeppos, as the name suggests, where we include many projects from the community that are relevant to Zepp OS.

We noticed that several of your projects are publicly available on Github and would like to ask your permission to include them in the official awesome zeppos repository.

Hopefully, we can get your approval and your project will help more developers in the community.

**Comment by Alpaca131** [comment]
Thank you for asking me.
Of course I'm totally fine with your request, but you should include this [repo](https://github.com/ZoLArk173/Authenticator) instead, which I also contributed, because I'm not currently developing this app. (due to the open data issue in Japan.)

In my country the timetables of the train are not publicized for programming use, and I need to pay a subscription fee to get that information.
I'm still struggling to find the data source and not sure when I can make this app.

**Comment by MyoungXUE** [comment]
> Thank you for inviting me. Of course I'm totally fine with your request, but you should include this [repo](https://github.com/ZoLArk173/Authenticator) instead, which I also contributed, because I'm not currently developing this app. (due to the open data issue in Japan.)
> 
> In my country the timetables of the train are not publicized for programming use, and I need to pay a subscription fee to get that information. I'm still struggling to find the data source and not sure when I can make this app.

Thank you very much, I will follow your idea.

**Comment by Alpaca131** [comment]
@MyoungXUE 
I found a way to make this app work legally. (without violating ToS of any company)
Could you include this app after I've finished the development?

**Comment by MyoungXUE** [comment]
@Alpaca131 Of course, all good work is always welcome, and I will add this app to the repository when you're done.
